atom_gems/AtomTutorials/Code/Include/AtomTutorials/AtomTutorialsBus.h

```diff
@@ -0,0 +1,31 @@
+
+#pragma once
```

Comment:
Since this bus is just boilerplate, and we don't have any future plans to use it for anything, let's remove this file.

Reply:
Removed the bus and removed the parts in the SystemComponent that were dependent on the bus & AZTickBus